packages/card/src/types.ts

```diff
@@ -7,6 +7,7 @@ export type FzCardProps = {
   contentClass?: string;
   collapsible?: boolean;
   defaultExpanded?: boolean;
+  alwaysAlive?: boolean;
```

Comment:
All props are missing documentation which we usually write through /** comments. It isn't clear what alwaysAlive in particular does. I'd add those and make sure that storybook correctly displays them.